pwa/src/components/deleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import "./deletemodal.css";
+import { Modal } from "@conductionnl/nl-design-system";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+interface DeleteModalProps {
+  handleDelete: any;
+  handleId: any;
+  optionalLength?: any;
+}
+
+const DeleteModal: React.FC<DeleteModalProps> = ({ handleDelete, handleId, optionalLength }) => {
+  return (
+    <div className="DeleteModal">
+      <Modal
+        title="Warning!"
+        id={`deleteModal${handleId}`}
+        body={function () {
+          return (
+            <>
+              {optionalLength === 1 ? (
+                <p>
+                  Are you sure you want to delete {handleId}? <br></br>
```

Comment:
You're using `<br/>` tags here to add a form of styling. I would recommend adding this styling to the `deleteModal.css`.